src/EventDelegate/index.ts

```diff
@@ -8,49 +8,99 @@ let closest = require('closest');
 
 export class EventDelegate {
   public init(): void {
-    EnumUtil.getNames(TriggerType).forEach((rawTriggerType: string) => {
-      let trigger: Trigger = new Trigger(rawTriggerType);
-      let eventType: string = <string>trigger.eventType;
 
-      document.removeEventListener(eventType, this.listener);
-      document.addEventListener(eventType, this.listener, trigger.useCapture);
-    });
-  }
+    let trigger:Trigger = new Trigger(TriggerType[TriggerType["CLICK"]]);
```

Comment:
do you think we could do something like this to clean up this code a little bit?

```javascript
private _setEventListener(trigger, listener): void {
   document.removeEventListener(<string>trigger.eventType, listener);
   document.addEventListener(<string>trigger.eventType, listener, trigger.useCapture);
}
public init(): void {
   let trigger:Trigger = new Trigger(TriggerType[TriggerType["CLICK"]]);
   this._setEventListener(trigger, this.onClick);
// repeat for all triggers
}
```